docker-compose.yml

```
@@ -0,0 +1,85 @@
+services:
+  
+  php:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: business_care_php
+    restart: unless-stopped
+    environment:
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)